packages/styles/src/popover.scss

```diff
@@ -6,13 +6,31 @@
 $block: #{$fd-namespace}-popover;
 $blockBody: #{$fd-namespace}-popover__body;
 
+body.overlay-active::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: var(--fdModal_BackgroundColor);
```

Comment:
we usually do `background: var(--fdModal_BackgroundColor);` in such cases. It does absolutely the same with a slight difference. 
`background` is a shortcut for:
 - background-color
 - background-image
 - background-repeat
 - background-attachment
 - background-position

So it will set the background-color and do exactly the same as `background-color: var(--fdModal_BackgroundColor);`. 

`background` overwrites all previous style. If you have:
```
background-image:url("..."); 
background-size:100% auto;
```

`background: red;` <- will replace all properties
`background-color: red;` <-will only replace bgcolor property